<launch>
    <arg name="odom_topic" default="odometry/kalman"/>
    <arg name="matching_err_tol" default="0.05"/>
    <arg name="inlier_fraction_tol" default="0.95"/>
    
    <node pkg="map_matching_localization" name="kalman" type="kalman.py" />
    <node pkg="map_matching_localization" name="relocalizer" type="relocalizer.py">
        <param name="odom_topic" value="$(arg odom_topic)"></param>
        <param name="matching_err_tol" value="$(arg matching_err_tol)"></param>
        <param name="inlier_fraction_tol" value="$(arg inlier_fraction_tol)"></param>
    </node>
</launch>